src/main/java/uk/gov/hmcts/ccd/domain/service/common/AccessControlService.java

```diff
@@ -329,8 +321,70 @@ private boolean hasCreateAccess(String newFieldName, final List<CaseField> caseF
     }
 
     private boolean valueDifferentAndHasUpdateAccess(JsonNode newData, JsonNode existingData, String newFieldName, final List<CaseField> caseFieldDefinitions, final Set<String> userRoles) {
-        return existingData.get(newFieldName).equals(newData.get(newFieldName))
-            || hasCaseFieldAccess(caseFieldDefinitions, userRoles, CAN_UPDATE, newFieldName);
+        if (existingData.get(newFieldName).equals(newData.get(newFieldName))) {
+            return true;
+        }
+        Optional<FieldType> fieldType = getCaseFieldType(caseFieldDefinitions, newFieldName);
+        if (!fieldType.isPresent()) {
+            return false;
+        }
+        if (!fieldType.get().getType().equals(COLLECTION)) {
+            return hasCaseFieldAccess(caseFieldDefinitions, userRoles, CAN_UPDATE, newFieldName);
+        } else {
+            if (!itemAddedAndHasCreateAccess(newData, newFieldName, caseFieldDefinitions, userRoles)) {
+                return false;
+            }
+            if (!itemDeletedAndHasDeleteAccess(existingData, newData, newFieldName, caseFieldDefinitions, userRoles)) {
+                return false;
+            }
+            return itemUpdatedAndHasUpdateAccess(existingData, newData, newFieldName, caseFieldDefinitions, userRoles);
+        }
+    }
+
+    private boolean itemDeletedAndHasDeleteAccess(JsonNode existingData, JsonNode newData, String newFieldName,
+                                                  List<CaseField> caseFieldDefinitions, Set<String> userRoles) {
+        JsonNode newValue = newData.get(newFieldName);
+        JsonNode oldValue = existingData.get(newFieldName);
+        boolean containsDeletedItem = StreamSupport.stream(spliteratorUnknownSize(oldValue.elements(), Spliterator.ORDERED), false).anyMatch(
+            oldItem -> {
+                boolean itemExists = StreamSupport.stream(spliteratorUnknownSize(newValue.elements(), Spliterator.ORDERED), false)
```

Comment:
Any way we can extract body and pass a function that does the comparison for the actual itemExists?